src/navigation/BottomTabNavigation/bottomTabNavigation.settings.tsx

```diff
@@ -0,0 +1,28 @@
+import { BottomTabNavigationOptions } from '@react-navigation/bottom-tabs';
+
+import { Icon } from 'components/Icon/Icon';
+
+import { COLORS } from 'constants/color';
+
+export const HOME_STACK_OPTIONS: BottomTabNavigationOptions = {
+  headerShown: false,
+  tabBarLabel: 'Chats',
+  tabBarStyle: {
+    backgroundColor: COLORS.light.primary[400],
+    borderTopWidth: 0,
+    elevation: 0,
+  },
+  tabBarLabelStyle: {
+    fontSize: 12,
+    fontWeight: 'bold',
+  },
+  tabBarIcon: ({ focused }) => {
+    return (
+      <Icon
+        color={focused ? COLORS.blue : COLORS.grey}
+        name="message"
+        size={22}
+      />
+    );
+  },
```

Comment:
Please make this a reusable arrow function, also add const for icon size and object with icon names for screens.

Example:
```
const ICON_SIZE = 22;

const ICON_NAME = {
  home: "message"
 };

 const renderIcon = (focused: boolean, name: string)  =>  (
      <Icon
        color={focused ? COLORS.blue : COLORS.grey}
        name={name}
        size={ICON_SIZE}
      />
    )
 ```